### PR TITLE
Icebox the Xeno update

### PR DESCRIPTION
**Pull request by nightred**
## About The Pull Request

This is an overhaul of the current Xeno setup in Icebox giving the RD a better office and expands maintenance tunnels.

<details><summary>The largest change is MultiZ.</summary>

image
</details>

The Xeno's Camera consoles have a separate camera network for fast camera swapping.
These cameras are still part of the normal camera network for both rd and station.

<details><summary>The RD got an update to their office.</summary>

image
</details>
The RD has a private observation deck for the test chamber, we have to keep the heads safe when doing science.

The test chamber has an observation area.
Basement loading area to keep things contained.
There is a Science exit to ice world to collect the interesting fauna for science.
A new layer of Maintenance surrounding the lower level with a few ladders for fun and games.
External path to Botany if you can brave the run.

<details><summary>Change pics:</summary>

image
image
</details>

## Why It's Good For The Game

Fun new area for xeno

## Changelog
:cl:
tweak: Icebox Xenobiology has been expanded in new directions
tweak: Icebox The Research Directors office has been updated
/:cl:

### CODE REVIEW

**Comment by Barhandar** [comment]
Doesn't this mean that slimes now can't effectively escape to station (simplemobs don't move over ladders and stairs when randomly moving around, AFAIK)?

**Comment by hannahscript** [comment]
I love love the catwalk with machinery under it aesthetic

**Comment by nightred** [comment]
Simple mobs not using stairs is likely a bug that needs to be looked at if true, I would not expect them to be able to use ladders.

**Comment by Tlaltecuhtli** [comment]
do the xenobio cams work from the above level?

**Comment by Dennok** [comment]
> do the xenobio cams work from the above level?

You can jusp to cam net on bottom level.

**Comment by nightred** [comment]
Yes they work, and because the xams have multi networks, and the consoles are set to the xeno pen cams network the consoles only see the xeno cams.

**Comment by san7890** [comment]
Where does the Toxins Gas Dump go to, because if I'm reading it right, it looks like it'll dump right onto the xenobiology building. I may be wrong, I'm just looking at your screenshots.

**Comment by nightred** [comment]
It should not, I will validate. That might be an oversight.

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 1]
```diff
@@ -50112,8 +49469,12 @@
 /turf/closed/wall/r_wall,
 /area/science/mixing/chamber)
 "fep" = (
-/obj/structure/window/reinforced,
-/turf/open/floor/engine,
+/obj/machinery/camera{
+	c_tag = "Xenobiology Pens Observastion - Starboard Aft";
```
Comment (Barhandar) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:49473]: Observation, same for all other instances of "Observa_s_tion"

Comment (nightred) [reply _maps/map_files/IceBoxStation/IceBoxStation.dmm:49473]: fixed up fixed up the wording

**Review by ghost — APPROVED** [review]
Very cool nightred

**Pull request merged**